### ISSUE TITLE
isEmpty function bug

### ISSUE THREAD

**Issue by 5kahoisaac** [rayepps/radash#95]
`isEmpty` function is not work for number.

Expected return `false`
Actual return `true`

Code from Git
![Screenshot 2022-10-06 at 10 58 08 AM](https://user-images.githubusercontent.com/49708270/194204046-9c2b6f20-b55a-45cf-8936-44bd5a38ca59.png)

Actual bundled code from npm package
![Screenshot 2022-10-06 at 10 55 26 AM](https://user-images.githubusercontent.com/49708270/194203762-fa43e560-1320-454e-a56b-8052d3e802cf.png)

Online Testing result
![Screenshot 2022-10-06 at 10 53 48 AM](https://user-images.githubusercontent.com/49708270/194203629-1820f3b5-b80b-4dea-ac26-301d0c903c03.png)

**Comment by matteosacchetto** [comment]
It has already been solved with commit 6d42fc7a107b6e47784dd90e5395a2697c664093
I can confirm that the version on the repo works as intended

```javascript
import { isEmpty } from "radash";

console.log(isEmpty(0)); // true
console.log(isEmpty(1)); // false
```

The version on NPM has not yet been update though

**Comment by rayepps** [comment]
Thanks for bringing this up with all the details @5kahoisaac ! It has been fixed on the main branch but not yet published (as @matteosacchetto pointed out), will be published this weekend with a lot of other little fixes.
